.github/workflows/tests.yml

```diff
@@ -34,15 +34,15 @@ jobs:
           - { python-version: "3.9",  tox-env: type-checking }
           - { python-version: "3.10", tox-env: type-checking }
           - { python-version: "3.11", tox-env: type-checking }
-          - { python-version: "3.8",  tox-env: py38,          max-attempts: 3 }
-          - { python-version: "3.8",  tox-env: py38-no-ext,   max-attempts: 3 }
+          - { python-version: "3.8",  tox-env: py38,          max-attempts: 1, timeout-minutes: 20}
```

Comment:
Note: This and other adjustments to tests.yml are going to be deleted once we have passing tests.